src/components/ReportActionItem/TripDetailsView.tsx

```diff
@@ -164,16 +169,66 @@ type TripDetailsViewProps = {
 function TripDetailsView({tripRoomReport, shouldShowHorizontalRule, tripTransactions}: TripDetailsViewProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+
+    const getTripDescription = useCallback(
+        (amount: number, currency: string, reservations: ReservationData[]) => {
+            const trips = `${reservations.length} ${reservations.length === 1 ? translate('travel.trip') : translate('travel.trips')}`;
+            return `${convertToDisplayString(amount, currency)} • ${trips.toLowerCase()}`;
+        },
+        [translate],
+    );
+
+    const getTripTitle = useCallback(
+        (reservations: ReservationData[]) => {
+            if (reservations.length === 0) {
+                return '';
+            }
+
+            const firstReservation = reservations.at(0)?.reservation;
+            const lastReservation = reservations.at(reservations.length - 1)?.reservation;
+
+            if (!lastReservation || !firstReservation) {
+                return '';
+            }
+
+            switch (firstReservation?.type) {
+                case CONST.RESERVATION_TYPE.FLIGHT: {
+                    const destinationReservation = reservations.filter((reservation) => reservation.reservation.legId === firstReservation.legId).at(-1);
+                    if (!destinationReservation) {
+                        return '';
+                    }
+                    return `${translate('travel.flightTo')} ${formatAirportInfo(destinationReservation.reservation.end, true)}`;
+                }
+                case CONST.RESERVATION_TYPE.TRAIN:
+                    if (reservations.length === 2 && firstReservation.start.shortName === lastReservation.end.shortName) {
+                        return `${translate('travel.trainTo')} ${Str.recapitalize(lastReservation.start.longName ?? '')}`;
+                    }
+                    return `${translate('travel.trainTo')} ${Str.recapitalize(lastReservation.end.longName ?? '')}`;
+                case CONST.RESERVATION_TYPE.HOTEL: {
+                    const nights = differenceInCalendarDays(new Date(lastReservation?.end.date), new Date(firstReservation.start.date));
+                    return `${nights} ${nights > 1 ? translate('travel.nightsIn') : translate('travel.nightIn')} ${Str.recapitalize(firstReservation.start.longName ?? '')}`;
+                }
+                case CONST.RESERVATION_TYPE.CAR: {
+                    const days = differenceInCalendarDays(new Date(lastReservation.end.date), new Date(firstReservation.start.date));
+                    return `${days} ${days > 1 ? translate('travel.daysCarRental') : translate('travel.dayCarRental')}`;
```

Comment:
If the user picks up the car and returns it the same day, would this return "0 day car rental" because we are using `differenceInCalendarDays`?

Reply:
This would be my fix btw
```suggestion
                    return days > 1 ? `${days} translate('travel.daysCarRental')` : `1 translate('travel.dayCarRental')}`;
```

Reply:
So the minimum is always "1 day car rental"